### PR TITLE
Feature/sof 6061

### PR DESCRIPTION
**Pull request by tjduigna**

### CODE REVIEW

File: schema/job/compute.json [hunk 1]
```diff
@@ -16,15 +16,29 @@
...
+            "type": "integer",
+            "validatePpn": {},
+            "errorMessage": "Max count exceeded"
         },
         "timeLimit": {
             "description": "Wallclock time limit for computing a job. Clock format: 'hh:mm:ss'",
-            "type": "string"
+            "type": "string",
+            "validateTimeLimit": {},
+            "errorMessage": "Time, 00:00:00 - 99:59:59"
```
Comment (azech-hqs) [inline schema/job/compute.json]: Do we only allow for wall time limits below 100h? Some calculations may take longer than 4 days.

File: schema/job/compute.json [hunk 2]
```diff
@@ -16,15 +16,29 @@
...
-            "type": "integer"
+            "type": "integer",
+            "validateNodes": {},
+            "errorMessage": "Max node count for selected queue exceeded"
         },
         "ppn": {
             "description": "number of CPUs used for the job inside the RMS.",
-            "type": "integer"
+            "type": "integer",
+            "validatePpn": {},
```
Comment (azech-hqs) [inline schema/job/compute.json]: I understand that the validation functions such as `validatePpn` are implemented in the web-app and `errorMessage` seems to be a special keyword by the `ajv-errors` extension. Should we note somewhere that these fields are not standard JSON schema attributes (e.g. bottom of readme.md)?

**Review by azech-hqs — COMMENTED** [review]
No objections

File: schema/job/compute.json (hunk 1 above)
Comment (tjduigna) [reply schema/job/compute.json]: This is a pre-existing validation logic but yes I agree.

Comment (timurbazhirov) [reply schema/job/compute.json]: `errorMessage` to be moved to JS

File: schema/job/compute.json (hunk 2 above)
Comment (timurbazhirov) [reply schema/job/compute.json]: For now, just adding explanation in the top-level `description` field (currently not present) would be sufficient.

**Review by timurbazhirov — APPROVED** [review]

**Review by timurbazhirov — APPROVED** [review]

**Pull request merged**